### PR TITLE
Add support for dedicated bus ways

### PR DESCRIPTION
**Pull request by nlehuby**
about #2

I've updated the overpass query to also get dedicated bus highways and display them.
image

I think some refactoring is needed for the display, because the right / left distinction is not really relevant for these kind of ways.
And I did not look at the editorMode yet.

### CODE REVIEW

File: planes.js [hunk 1]
```diff
@@ -521,7 +531,7 @@ function getQueryParkingLanes() {
         var bbox = [bounds.getSouth(), bounds.getWest(), bounds.getNorth(), bounds.getEast()].join(',');
         return editorMode
             ? '[out:xml];(way[highway~"^motorway|trunk|primary|secondary|tertiary|unclassified|residential"](' + bbox + ');)->.a;(.a;.a >;.a <;);out meta;'
-            : '[out:xml];(way[highway][~"^(lanes:(psv|bus)|busway).*"~"."](' + bbox + ');)->.a;(.a;.a >;);out meta;';
+            : '[out:xml];(way["highway"][~"^(lanes:(psv|bus)|busway).*"~"."](' + bbox + ');way["highway"][~"psv|bus"~"yes"](' + bbox + ');)->.a;(.a;.a >;);out meta;';
```
Comment (zlant) [inline planes.js]: `way["highway"][access=no][~"psv|bus"~"yes"]`
add [access=no]

**Comment by nlehuby** [comment]
image
small update with dedicated ways displayed in green.

**Comment by zlant** [comment]
Change target branch to `feature/bus-ways` please

**Comment by nlehuby** [comment]
I've switched the base branch and push a first working version with the editor too.

It still have some issues with the offset of the dedicated road:
image
image

**Comment by nlehuby** [comment]
[#16/48.8437/2.3677](http://localhost:8080/#16/48.8433/2.3680) is a good place for testing: there are both bus lanes and dedicated roads

**Pull request merged**